TestApp/update-npm-packages.sh

```diff
@@ -1,4 +1,33 @@
 #!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
remove copyright. no reason to add it to an existing file

Reply:
done